client/me/help/help-contact/index.jsx

```diff
@@ -211,16 +210,14 @@ module.exports = React.createClass( {
 	},
 
 	onOperatorsAway: function() {
-		const { isOlarkReady, isUserEligible, details } = this.state.olark;
-		const showChatVariation = isUserEligible && details.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( details.isConversing || isUserEligible );
-		const showForumsVariation = ! ( showChatVariation || showKayakoVariation );
+		const { isUserEligible, details } = this.state.olark;
 
 		if ( ! details.isConversing ) {
 			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
-				form_type: showKayakoVariation ? 'kayako' : 'forum'
+				form_type: isUserEligible ? 'kayako' : 'forum'
```

Comment:
After staring at this long enough I realize that the "forum" case should theoretically never happen. If a user is not eligible for chat and they aren't currently chatting then they'd never experience an "offline" contact form. The form would always just be the "Ask in the forums" variation.